### ISSUE TITLE
The security accessibility of the rewriting method must be matched with the security of the method being rewritten.

### ISSUE THREAD

**Issue by wuchao288** [phatnguyen81/TinyCms2#1]
Override the member "Autofac.Integration.Mvc.AutofacDependencyResolver.GetService (System.Type)" in violation of the rules of succession security. The security accessibility of the rewriting method must be matched with the security of the method being rewritten.
